src/njoy21/input/DTFR/Card2/Iedit.hpp

```diff
@@ -0,0 +1,18 @@
+struct Iedit {
+  using Value_t = int;
+  static std::string name(){ return "iedit"; }
+
+  static std::string description(){
+    return
+      "The iedit value is for edit control, and can either equal 0 or 1.\n"
+      "A value of 0 correspond to table-edits option, while a value of 1\n"
```

Comment:
iedit = 1 corresponds to the separate-table options, as stated in line 9. Should I try to go into more detail?